app/views/admin/loans/timeline/_main.html.slim

```diff
@@ -27,3 +27,12 @@ section.timeline-table
 
   ul.dropdown-menu.dropdown-menu-right id="project-step-menu"
     li: a href="#" data-action="edit" = t(:edit)
+    li: a href="#" data-action="duplicate" = t("project_step.duplicate")
+    li
+      a [href="#" data-action="delete"
+        data-confirm== t("project_step.confirm_deletion")
```

Comment:
The double = signs are not necessary—slim automatically treats attribute values as ruby code.